README.md

```diff
@@ -142,4 +142,5 @@ The following debug environment variables are available for use with the OpenCL
 |:---------------------------------:|---------------------|----------------------|
 | OCL_ICD_FILENAMES                 | Specifies a list of additional ICDs to load.  The ICDs will be enumerated first, before any ICDs discovered via default mechanisms. | `export OCL_ICD_FILENAMES=libVendorA.so:libVendorB.so`<br/><br/>`set OCL_ICD_FILENAMES=vendor_a.dll;vendor_b.dll` |
 | OCL_ICD_VENDORS                   | On Linux and Android, specifies a directory to scan for ICDs to enumerate in place of the default `/etc/OpenCL/vendors'. |  `export OCL_ICD_VENDORS=/my/local/icd/search/path` |
-| OPENCL_LAYERS                    | Specifies a list of layers to load. |  `export OPENCL_LAYERS=libLayerA.so:libLayerB.so`<br/><br/>`set OPENCL_LAYERS=libLayerA.dll;libLayerB.dll` |
+| OPENCL_LAYERS                     | Specifies a list of layers to load. |  `export OPENCL_LAYERS=libLayerA.so:libLayerB.so`<br/><br/>`set OPENCL_LAYERS=libLayerA.dll;libLayerB.dll` |
+| OCL_LAYERS                        | On Linux and Android, specifies a directory to scan for layers to enumerate in place of the default `/etc/OpenCL/layers'. | `export OCL_ICD_VENDORS=/my/local/layers/search/path` |
```

Comment:
OCL_LAYERS vs OPENCL_LAYERS are going to be confusing to remember. Maybe OCL_LAYER_LIBS and OCL_LAYER_DIR (assuming it's not too late to change the existing name)?

Reply:
Good catch on the copy paste, I'll fix that.
For the naming of the environment variables, you are correct that it is going to be difficult to converge on something satisfactory. The original name of OPENCL_LAYERS was OCL_ICD_LAYERS, but was changed by request of the working group. Unfortunately, rolling it back/changing it is not possible anymore. We can still change OCL_LAYERS to OCL_LAYER_LIBS or OPENCL_LAYER_LIBS.

Reply:
I implemented the change @bashbaug suggested, as it seems a good choice.